### PR TITLE
Added favicon

### PR DESCRIPTION
**Pull request by SachinRammoorthy**
Added favicon to the site.

Issue #75

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,9 +15,8 @@
     <link rel="stylesheet" href="stylesheets/contributors.css">
 
     <link href="https://fonts.googleapis.com/css?family=Raleway|Lato:400,900|Quicksand|Open+Sans|Roboto+Mono" rel="stylesheet">
-    <!--[if lt IE 9]>
-        <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
+    <link rel="shortcut icon" type="image/png" href="/images/logos/jboss-logo.png"/>
+
```
Comment (aashutoshrathi) [inline index.html]: Remove this redundant blank line

File: index.html [hunk 2]
```diff
@@ -15,9 +15,8 @@
     <link rel="stylesheet" href="stylesheets/contributors.css">
 
     <link href="https://fonts.googleapis.com/css?family=Raleway|Lato:400,900|Quicksand|Open+Sans|Roboto+Mono" rel="stylesheet">
-    <!--[if lt IE 9]>
-        <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
+    <link rel="shortcut icon" type="image/png" href="/images/logos/jboss-logo.png"/>
```
Comment (aashutoshrathi) [inline index.html]: Preferred Favicon extension for all browser is `.ico`, please switch to it

Comment (aashutoshrathi) [inline index.html]: Also, preferred relation is `icon`

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi worked on changes.

**Comment by aashutoshrathi** [comment]
@SachinRammoorthy Please remove the png file and I have already told you, it is not good practice sending PR from master branch, please understand and send a new PR for the same from a different branch than the master.

**Comment by SachinRammoorthy** [comment]
Shifted to #80 with new branch :)

**Pull request closed without merging**